packages/beacon-node/src/api/impl/validator/index.ts

```diff
@@ -664,14 +539,14 @@ export function getValidatorApi({
 
   return {
     produceBlock,
-    produceBlockV2: produceFullBlockOrContents,
+    produceBlockV2,
```

Comment:
hold off on any produceBlock changes except the waitForSlot changes that you are going for here, there are changes stacked on followup PRs which will get conflicted with this anyway